docs/language/values-and-types/strings-and-characters.md

```diff
@@ -83,151 +83,142 @@ let arr: String = "\(x)"
+  ```
 
-    ```cadence
-    let example = "Flowers"
+- ```cadence
+  view fun toLower(): String
+  ```
 
-    example.toLower()  // is `flowers`
-    ```
+  Returns a string where all upper case letters are replaced with lowercase characters.
 
--
-    ```cadence
-    view fun replaceAll(of: String, with: String): String
-    ```
+  ```cadence
+  let example = "Flowers"
 
-    Returns a string where all occurences of `of` are replaced with `with`. If `of` is empty, it matches at the beginning of the string and after each UTF-8 sequence yielding k+1 replacements for a string of length k.
+  example.toLower()  // is `flowers`
+  ```
 
-    ```cadence
-    let example = "abababa"
+- ```cadence
+  view fun replaceAll(of: String, with: String): String
+  ```
 
-    example.replaceAll(of: "a", with: "o")  // is `obobobo`
-    ```
+  Returns a string where all occurences of `of` are replaced with `with`. If `of` is empty, it matches at the beginning of the string and after each UTF-8 sequence yielding k+1 replacements for a string of length k.
 
--
-    ```cadence
-    view fun split(separator: String): [String]
-    ```
+  ```cadence
+  let example = "abababa"
 
-    Returns the variable-sized array of strings created splitting the receiver string on the `separator`.
+  example.replaceAll(of: "a", with: "o")  // is `obobobo`
+  ```
 
-    ```cadence
-    let example = "hello world"
+- ```cadence
+  view fun split(separator: String): [String]
+  ```
 
-    example.split(separator: " ")  // is `["hello", "world"]`
-    ```
+  Returns the variable-sized array of strings created splitting the receiver string on the `separator`.
 
+  ```cadence
+  let example = "hello world"
+
+  example.split(separator: " ")  // is `["hello", "world"]`
+  ```
 
 The `String` type also provides the following functions:
 
--
-    ```cadence
-    view fun String.encodeHex(_ data: [UInt8]): String
-    ```
+- ```cadence
+  view fun String.encodeHex(_ data: [UInt8]): String
+  ```
 
-    Returns a hexadecimal string for the given byte array
+  Returns a hexadecimal string for the given byte array
 
-    ```cadence
-    let data = [1 as UInt8, 2, 3, 0xCA, 0xDE]
+  ```cadence
+  let data = [1 as UInt8, 2, 3, 0xCA, 0xDE]
 
-    String.encodeHex(data)  // is `"010203cade"`
-    ```
+  String.encodeHex(data)  // is `"010203cade"`
+  ```
 
--
-    ```cadence
-    view fun String.join(_ strings: [String], separator: String): String
-    ```
+- ```cadence
+  view fun String.join(_ strings: [String], separator: String): String
+  ```
 
-    Returns the string created by joining the array of `strings` with the provided `separator`.
+  Returns the string created by joining the array of `strings` with the provided `separator`.
 
-    ```cadence
-    let strings = ["hello", "world"]
-    String.join(strings, " ") // is "hello world"
-    ```
+  ```cadence
+  let strings = ["hello", "world"]
+  String.join(strings, " ") // is "hello world"
```

Comment:
This example is actually missing the required argument label of the second parameter:

```suggestion
  String.join(strings, separator: " ") // is "hello world"
```